tomviz/pybind11/eigenConversion.h

```diff
@@ -0,0 +1,770 @@
+/*
+    Transparent conversion for dense and sparse Eigen matrices
+
+    This function is adapted from the pybind11 code by Wenzel Jacob.
+    pybind11 - https://github.com/pybind/pybind11
+
+    Copyright (c) 2016 Wenzel Jakob <wenzel.jakob@epfl.ch>
+
+    All rights reserved. Use of this source code is governed by a
+    BSD-style license that can be found in the LICENSE file.
+*/
```

Comment:
Same question, and what is BSD-style? Is has to be BSD or it can't be merged.

Reply:
@cjh1 @cryo this file is from the pybind11 library: https://github.com/pybind/pybind11/blob/master/include/pybind11/eigen.h the original script imports Eigen directly while we only have access to the VTK - wrapped version. So I had to change the include statements so it references <vtk_eigen.h> instead of eigen. This should be ok, correct?